bin/run_unittests.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+export PYTHONPATH
```

Comment:
just export PYTHONPATH does nothing because this variable is empty here.
You can use:
export ${PYTHONPATH-:`pwd`/src}

`python3 unittests/simple_commander.py ...`